Slim/Plugin/WiMP/strings.txt

```diff
@@ -1,10 +1,12 @@
-PLUGIN_WIMP_MODULE_NAME
+﻿PLUGIN_WIMP_MODULE_NAME
 	EN	TIDAL
 	NL	TIDAL
+	NO	TIDAL
```

Comment:
No need to repeat strings which are identical with the English term. Just leave it alone - LMS would automatically use the identical English version as a fallback. Saving a few bytes :-)